src/documentation/read_ntfs_fares.md

```diff
@@ -0,0 +1,32 @@
+# Reading NTFS fares
+## Introduction
+This document describes how [NTFS fares](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fare_extension_fr.md) are loaded in Navitia Transit Model.
+
+In this initial version: 
+- only tickets on origin-destination stops are taken into account
+- the NTFS file *fares.csv* is ignored, as the NTM fares model does not yet support special conditions on tickets (e.g validity duration, number of transfers allowed, constraints on networks/lines) 
+- tickets specified in a currency different than EUR are ignored
+- only tickets on origin-destination stops are taken into account
+
+### Loading Tickets
+
+NTM property | NTFS file | NTFS field | Note/mapping rule
+--- | --- | --- | ---
+id | prices.csv | \*clef de ticket\* | 
+start_date | prices.csv | \*date de début de validité\* | 
+end_date | prices.csv | \*date de fin de validité\* | The previous date of the specified date in the input.
+currency_type | prices.csv | \*devise\* | The currency is set to `EUR` when the input value is `centime`.
+price | prices.csv | \*prix\* | The specified input value is converted into EUR.
+validity_duration | | | This field is ignored.
+transfers | | | This field is ignored.
+
+### Loading OD Rules
+
+NTM property | NTFS file | NTFS field | Note/mapping rule
+--- | --- | --- | ---
+id | prices.csv | \*clef de ticket\* | The id is prefixes with `OD:`.
```

Comment:
Not sure: isn't there a mixup on those 2 first rows?

Reply:
Actually, `id` corresponds to the id of the rule (which is formed based on the input ticket id and a prefix).
`ticket_id` corresponds to the id of the ticket with the OD (and it is based on the input ticket id).